core/src/main/java/com/scalar/db/io/Value.java

```diff
@@ -5,21 +5,26 @@
 import javax.annotation.Nonnull;
 
 /**
- * An abstraction for storage entry's value
+ * An abstraction for storage entry's value (column).
+ *
+ * <p>This class and the implementation classes of it are primarily for internal use. Breaking
+ * changes can and will be introduced to them. Users should not depend on them. And maybe we will
+ * rename this class to {@code Column} in the future because it is a more proper name for this
+ * abstraction.
```

Comment:
I documented that this method is for internal use and we might be rename it to `Column` in the future.

I just changed Javadoc for this `Value` class and the implementation classes (`xxxValue`).